examples/complete/main.tf

```diff
@@ -1,3 +1,164 @@
 ##############################################################################
 # Complete example
 ##############################################################################
+
```

Comment:
can you update the `examples/complete/README.md` with details of what this example does? If it is a more advanced example than the basic example, perhaps rename the example "Advanced example" or something else relevant to how the example differs from the basic example

Reply:
It adds an load balancer to the environment to attach the scaled instances.  I updated the README to include the resources it provisions but let me know if you want me to change the name